src/services/sync-checker.ts

```diff
@@ -539,6 +532,13 @@ export class SyncChecker {
       pocketConfiguration.rejectSelfSignedCertificates
     )
   }
+
+  parseBlockFromPayload(payload: object, syncCheckResultKey: string): number {
+    const rawHeight = payload[`${syncCheckResultKey}`]
```

Comment:
This does not work with chains which result is in a nested object, right now the current chains does not have the issue, but could add a TODO atm for when's needed